libfwupdplugin/fu-dpaux-device.c

```diff
@@ -128,6 +128,33 @@ fu_dpaux_device_setup(FuDevice *device, GError **error)
 	priv->dpcd_hw_rev = fu_struct_dpaux_dpcd_get_hw_rev(st);
 	priv->dpcd_dev_id = fu_struct_dpaux_dpcd_get_dev_id(st);
 	fu_device_set_version_raw(device, fu_struct_dpaux_dpcd_get_fw_ver(st));
+
+	/* build some extra GUIDs */
+	fu_device_add_instance_u32(device, "OUI", priv->dpcd_ieee_oui);
+	fu_device_add_instance_u8(device, "HWREV", priv->dpcd_hw_rev);
+	fu_device_add_instance_strup(device, "DEVID", priv->dpcd_dev_id);
+	fu_device_build_instance_id_full(device,
+					 FU_DEVICE_INSTANCE_FLAG_QUIRKS,
+					 NULL,
+					 "DPAUX",
+					 "OUI",
+					 NULL);
+	fu_device_build_instance_id_full(device,
+					 FU_DEVICE_INSTANCE_FLAG_QUIRKS,
+					 NULL,
+					 "DPAUX",
+					 "OUI",
+					 "HWREV",
+					 NULL);
+	fu_device_build_instance_id_full(device,
+					 FU_DEVICE_INSTANCE_FLAG_QUIRKS,
+					 NULL,
+					 "DPAUX",
+					 "OUI",
+					 "DEVID",
+					 NULL);
```

Comment:
presumably all the DP aux plugins README should mention this too now

Reply:
Do you think that we want a DPAUX\OUI+DEVID+HWREV combined one too?

Reply:
Both bits added.